draft-ietf-rats-architecture.md

```diff
@@ -194,6 +193,12 @@ provides device identity and protected storage for measurements.
 These components perform a series of measurements, and express this with Evidence as to the
 hardware and firmware/software that is running.
 
+~~~
+    FIXME from Henk: Measurements at early stages of
+    Layered Attestation are NOT evidence yet.
+    This text does not cover that yet
+~~~
```

Comment:
We should not be adding FIXME comments into the doc at this point

Reply:
This change has been reverted.